### DISCUSSION TITLE
Hide Sketchware Class Name

### DISCUSSION THREAD

**Discussion by AndroXStudiO** [Sketchware-Pro/Sketchware-Pro#550] [Q&A]
How to Hide and Delete Sketchware Class (Sketchware Util & SketchApplication) 
Name From Your App so no one can know which app or programme you used !!!

**Comment by j-px** [comment]
You could use ProGuard for that. Also, setting a custom application class name should already stop generating SketchApplication.java

**Reply by rukamori** [reply]
isn't it an error when we change the class name? i have tried to change it in my project settings but it makes the project i build can't be opened /fc

**Reply by j-px** [reply]
Changing that only changes the value in AndroidManifest.xml. You need to create a custom application class in Java Manager with the same name for it to work. For example, when you enter `.Application` in Project Configuration, you have to create `Application.java` in Java Manager.
